.github/workflows/integration.yml

```diff
@@ -0,0 +1,81 @@
+name: E2E test
+on: [push]
+jobs:
+  dbzui-container:
+    name: Cypress
+    runs-on: ubuntu-latest
+
+    services:
+      dbzui-zookeeper:
+        image: debezium/zookeeper:1.5
+
+      dbzui-kafka:
+        image: debezium/kafka:1.5
+        ports:
+          - 9092:9092
+        env: 
+          ZOOKEEPER_CONNECT: dbzui-zookeeper:2181
+
+      dbzui-postgres:
+        image: debezium/example-postgres:1.5
+        ports:
+          - 65432:5432
+        env: 
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+      
+      dbzui-connect:
+        image: debezium/connect:1.5
+        ports:
+          - 8083:8083
+        env:
+          BOOTSTRAP_SERVERS: dbzui-kafka:9092
+          GROUP_ID: 1
+          CONFIG_STORAGE_TOPIC: my_connect_configs
+          OFFSET_STORAGE_TOPIC: my_connect_offsets
+          STATUS_STORAGE_TOPIC: my_connect_statuses   
+
+    steps:
+      - name: Checkout Debezium UI
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Cache local Maven repository
```

Comment:
Same here. When you don't use Maven, you don't need to cache its repo.
Instead we should file an issue about adding caching of the docker build cache and intermediate images.